tests/test_barman_cloud_backup_delete.py

```diff
@@ -103,12 +103,57 @@ def _get_mock_backup_files(self, file_paths):
             for name, path in file_paths.items()
         )
 
+    def _create_backup_info(
+        self,
+        backup_id,
+        begin_wal=None,
+        end_wal=None,
+        backup_name=None,
+        parent_backup_id=None,
+        children_backup_ids=None,
+        is_incremental=False,
+        has_children=False,
+    ):
+        """
+        Helper for tests which creates mock BackupInfo objects.
+
+        If begin_wal has an entry for a backup_id then the begin_wal and
+        timeline values will be set according to the wal name.
```

Comment:
```suggestion
        If *begin_wal* is set then the backup's begin_wal and
        timeline values will be set according to the wal name.
```

It is no longer a dict heh